reconcile/slack_sender.py

```diff
@@ -0,0 +1,77 @@
+import sys
+import logging
+
+import reconcile.queries as queries
+from utils.slack_api import SlackApi
+
+from utils.state import State
+
+QONTRACT_INTEGRATION = 'slack-sender'
+
+
+def collect_to(to):
+    """Collect audience to send notification to from to object
+
+    Arguments:
+        to {dict} -- AppInterfaceNotification_v1 object
+
+    Raises:
+        AttributeError: Unknown alias
+
+    Returns:
+        set -- Audience to send notification to
+    """
+    audience = set()
+
+    users = to.get('users')
+    if users:
+        for user in users:
+            audience.add(user['org_username'])
+
+    channels = to.get('channels')
+    if channels:
+        for channel in channels:
+            audience.add(channel)
+
+    return audience
+
+
+def run(dry_run=False):
+    slackapi = SlackApi(username="app-sre-bot", icon_emoji=":eyes:")
```

Comment:
i would propose that we initiate SlackApi using the method defined in reconcile/slack_base.py.
this would also force you to consume slack from the query as defined in other integrations, which is desirable.